db/create_tables.sql

```diff
@@ -39,7 +39,7 @@ CREATE TABLE IF NOT EXISTS `events` (
   `type` VARCHAR(100) NOT NULL,
   `status` VARCHAR(100) DEFAULT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
```

Comment:
my bad, we need it. When the event is fired it would in pending state, and once it is consumed successfully it would go triggered status. That's what I inferred from Event class status column. During update we can use event id to identify the particular event.